static/app.js

```diff
@@ -154,27 +193,34 @@ $(function() {
             }  
         }
         else {
-            $('.results').empty()
-            term = $('#term').val()
-            try {
-                const response = await axios({
-                    url: `${base_url}search/`,
-                    method: "GET",
-                    params: {
-                        query: term
-                    }
-                })
-                Object.entries(response.data).forEach((entry) => {
-                    const [key, value] = entry;
-                    const brewery = value
-                    renderBrewery(brewery)
-                })
+             searchByKeyword()
             }   
-            catch(e) {
-                console.log(e.message)
-            }
-        }    
-    })
+    }   
+
+    async function searchByKeyword() {
+        $('.results').empty()
+        $('#recent-reviews').addClass('d-none')
+        term = $('#term').val()
+        try {
+            const response = await axios({
+                url: `${base_url}search/`,
+                method: "GET",
+                params: {
+                    query: term
+                }
+            })
+            Object.entries(response.data).forEach((entry) => {
```

Comment:
You can use Object.values here if not using the `key`

Reply:
done. 

 /** helper function to process brewery API response data
     * takes response data and dataType as parameters, turns response data into an results object to be rendered
     * dataType is in regards to autocomplete or normal results
     */

    function processBreweries(data, dataType){
        const breweriesResults = Object.values(data).map((brewery) => {
            return createBrewery(brewery, dataType)
        })
        return breweriesResults;
    }